From 6d3970658de8f270b651e8b4cffbc464e3774428 Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 2 Dec 2011 11:32:04 +0100
Subject: [PATCH] rc-local: no need to check if the script is executable

rc-local.service is pulled in by a generator only if the script is
executable. No need to check again.
(cherry picked from commit e951701a4d90152447e195e926e0e12c7dcc0051)
---
 units/fedora/rc-local.service |    1 -
 units/suse/rc-local.service   |    1 -
 2 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/units/fedora/rc-local.service b/units/fedora/rc-local.service
index fade90c..36100df 100644
--- a/units/fedora/rc-local.service
+++ b/units/fedora/rc-local.service
@@ -9,7 +9,6 @@
 # systemd-rc-local-generator if /etc/rc.d/rc.local is executable.
 [Unit]
 Description=/etc/rc.d/rc.local Compatibility
-ConditionFileIsExecutable=/etc/rc.d/rc.local
 
 [Service]
 Type=forking
diff --git a/units/suse/rc-local.service b/units/suse/rc-local.service
index df9a09b..11dc581 100644
--- a/units/suse/rc-local.service
+++ b/units/suse/rc-local.service
@@ -9,7 +9,6 @@
 # systemd-rc-local-generator if /etc/init.d/boot.local is executable.
 [Unit]
 Description=/etc/init.d/boot.local Compatibility
-ConditionFileIsExecutable=/etc/init.d/boot.local
 
 [Service]
 Type=oneshot
